### DISCUSSION TITLE
Where did virtual green screen go?

### DISCUSSION THREAD

**Discussion by Belidos3D** [Xaymar/obs-stream-effects#958] [Q&A] [answered]
What happened to the virtual greenscreen? 

I just updated to OBS 2,8 and updated the latest version of Stream FX and Virtual greenscreen has disappeared from the filter options. 

There is an option for NVidia Background Removal, but it doesn't seem to actually do anything, despite me having an RTX 3080.

**Comment by DigitalxGoon** [comment]
I have the same issue, except now I do not have the option for Nvidia Background removal either. I supposed this is because I rolled back my version of OBS to 27.2.4, because I could not get StreamFX to work in 28.0.3.

**Reply by Belidos3D** [reply]
Yeah, Nvidia Background Removal is a new feature in OBS 28. Which is maybe why StreamFX discarded Virtual Greenscreen. But to be quite frank, i finally got Nv's BG remover to work (you need to download a separate tool from NVidia) and it is absolutely awful, you can't get a good clean removal with it no matter how hard you try. Virtual Greenscreen on the other hand was amazing, the edges were so crisp and clear and it hardly ever produced artefacts.

**Reply by Xaymar** [reply]
This is the first time I've heard of this.

**Reply by DigitalxGoon** [reply]
I stumbled across it by mistake yesterday. I’ve been doing a lot of trouble shooting at home for a bitrate issue I’m having with OBS and one of the steps I took was uninstalling every single one of my plugins, as well as OBS and reinstalling. Prior to that, StreamFX was working just fine with OBS 28.0.3. 

Now, I’m using the Alpha for StreamFX (yeah I know it’s not recommended) and all the features are technically back but Virtual Greenscreen is now replace with Nvidia Background Removal. Agree with Belidos3D, VGS was a LOT better than the OBS built in implementation of Nvidia Background Removal.

**Comment by Xaymar** [comment, marked as answer]
Virtual Greenscreen is an experimental feature, and should still be present if the NVIDIA Video Effects SDK Redistributable is installed. Unfortunately, OBS Studio 28.0 is still not supported, as per massive text in the Release notes.

**Reply by DigitalxGoon** [reply]
Oh I know. But for some reason after a fresh install, even rolling back to the last working version it’s completely absent. The rest of the filters where still available, however, I couldn’t actually use them. As a note from my previous comment, the Video SDK was the ONLY thing I did NOT uninstall yesterday.

**Reply by Xaymar** [reply]
[What does 🔴, 🟠 and 🟢 mean on a Feature?](https://github.com/Xaymar/obs-StreamFX/wiki/Frequently-Asked-Questions#what-does---and--mean-on-a-feature)

**Comment by Belidos3D** [comment]
Where do i find release notes?

It would be great if VGS could be made compatible with 28.0, it's far better quality than the nvidia one and you don't need to have bloatware (nvidia broadcast) loaded to run it like the nvidia one.

**Reply by Xaymar** [reply]
The release notes are on every release, as you might expect them to be. For example, just click [this link](https://github.com/Xaymar/obs-StreamFX/releases/tag/0.11.1) for the release notes for 0.11.1.
